src/osd/scrubber/scrub_backend.h

```diff
@@ -46,15 +46,17 @@
 #include <string_view>
 
 #include "common/LogClient.h"
+#include "osd/OSDMap.h"
 #include "common/scrub_types.h"
+#include "osd/scrubber_common.h"
 
 struct ScrubMap;
 
 class PG;
 class PgScrubber;
-class PGBackend;
-class PGPool;
-
+//class PGBackend;
+struct PGPool;
+using Scrub::PgScrubBeListener;
```

Comment:
I'm not sure why this needs to be separate from ScrubBeListener -- they only seem to be consumed by ScrubBackend.

Reply:
But supplied by two different entities.